### ISSUE TITLE
The getDemonstrators function in the script returns an error

### ISSUE THREAD

**Issue by tmontanaro** [envri-fair/knowledge-base-ui#1]
When the page (https://envri-fair.github.io/knowledge-base-ui/) is loaded an error is returned with the following message "failundefined".

By analyzing the code, within the getDemonstrators function present in the js/function.js script,
the "data" variable is undefined after the execution of the query:

`"prefix envri: <http://envri.eu/ns/> prefix dcterms: <http://purl.org/dc/terms/> prefix schema: <http://schema.org/> prefix skos: <http://www.w3.org/2004/02/skos/core#> prefix fair: <https://w3id.org/fair/principles/terms/> select distinct ?principle_label ?principle_definition ?fair_principle ?demonstrator_label ?creator ?created ?demonstrator_url where { ?fair_principle a fair:FAIR-SubPrinciple . ?fair_principle skos:definition ?principle_definition . ?fair_principle rdfs:label ?principle_label . ?fair_principle envri:hasDemonstrator ?demonstrator . ?demonstrator a envri:TechnologyDemonstrator . ?demonstrator rdfs:label ?demonstrator_label . ?demonstrator dcterms:creator ?creator . ?demonstrator dcterms:created ?created . ?demonstrator schema:url ?demonstrator_url . FILTER (lang(?principle_label) = 'en' && lang(?principle_definition) = 'en') }"`

**Comment by markusstocker** [comment]
I suspect the SPARQL endpoint is no longer running. Anyhow, the knowledge-base-ui is an older version, which we then substituted with the following: https://envri-fair.github.io/fair-dashboard/

**Comment by markusstocker** [comment]
Indeed, I checked in the code and the SPARQL endpoint was https://envri-fair.lab.uvalight.net/sparql but this is no longer available.

**Comment by tmontanaro** [comment]
Thank you very much for your fast reply! I will the issue opened although it is actually solved in the new repo just to let other people know that you are already aware about the problem
